src/__swaps__/screens/Swap/components/CoinRow.tsx

```diff
@@ -15,59 +13,56 @@ import { setClipboard } from '@/hooks/useClipboard';
 import { RainbowNetworks } from '@/networks';
 import * as i18n from '@/languages';
 import { ETH_ADDRESS } from '@/references';
-import { trimTrailingZeros } from '@/__swaps__/utils/swaps';
 import { ParsedSearchAsset } from '@/__swaps__/types/assets';
 import { userAssetsStore } from '@/state/assets/userAssets';
 import { SearchAsset } from '@/__swaps__/types/search';
 import { ChainId } from '@/__swaps__/types/chains';
 
+export const COIN_ROW_WITH_PADDING_HEIGHT = 56;
```

Comment:
move this to constants.ts

Reply:
It's very tightly coupled to this component, what's the benefit of moving it?

Reply:
You are accessing it in both of the lists.

Reply:
I mean very tightly coupled in the sense that it's directly relevant to the layout of this component. Don't see the benefit of obscuring it within the context of the `CoinRow.tsx` file.